docs/src/examples/components/Tooltip/Types/TooltipExamplePointing.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+import { Button, Tooltip } from '@stardust-ui/react'
+
+const TooltipExamplePointing = () => (
+  <Tooltip open pointing content={`The tooltip is pointing.`}>
```

Comment:
```suggestion
  <Tooltip open pointing content="The tooltip is pointing.">
```

I will activate lint rule for this soon 😺